src/server/routes/entity/author.js

```diff
@@ -120,7 +120,6 @@ function authorToFormState(author) {
 
 	const buttonBar = {
 		aliasEditorVisible: false,
-		disambiguationVisible: Boolean(author.disambiguation),
```

Comment:
You removed this `disambiguationVisible` for author and work entites, but not for edition, edition-group and publisher

Reply:
@MonkeyDo 
hey! 
actually i commented them out , and forgot to erase the comments ,sorry for that .
now my exams are over and i will be correcting it from now .